spec/spec_helper.rb

```diff
@@ -23,6 +23,7 @@
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 # include the manageiq-gems-pending matchers
 Dir[ManageIQ::Gems::Pending.root.join("spec/support/custom_matchers/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
```

Comment:
Ugh, you're right, this is not needed. Instead of putting it just into https://github.com/ManageIQ/manageiq-ui-classic/pull/164 I mistakenly put it both into that & this PR. Thanks for noticing that.